src/main/java/dk/bec/polonez/reservationsystem/service/AuthService.java

```diff
@@ -0,0 +1,33 @@
+package dk.bec.polonez.reservationsystem.service;
+
+import dk.bec.polonez.reservationsystem.model.User;
+import dk.bec.polonez.reservationsystem.repository.UserRepository;
+import org.springframework.http.HttpStatus;
```

Comment:
**advice** Architectural remark: ```AuthService``` is service class (not controller class), I guess this should be a class with pure authentication business logic so it should be HTTP-protocol-agnostic but currently you imported and used two HTTP-specific or web-specific classes: ```HttpStatus``` and ```ResponseStatusException```.
Woldn't it be a better solution to throw pure business exception in line 30 (see my example below) and then implement a dedicated exception-mapper (that would map exception to HTTP response code) in a separate class (and in different package - presumably in ```controller```)?

Example:

```
return userRepository.findByUsername(username)
                .orElseThrow(() -> new UserForbiddenException());
```